### PR TITLE
update Filter to include pageSize and sort

### PR DESCRIPTION
**Pull request by chauger**
Fixed Bug with pageSize not staying after search

### CODE REVIEW

File: src/components/Filter.js [hunk 1]
```diff
@@ -39,8 +39,9 @@ const SubmitOnEnter = ({ children }) => {
 
 export default ({ children, defaultValues, pageSize, sorts, transform }) => {
   const [urlState, updateUrl] = useUrlParamState()
+  const process = useCallback(values => updateUrl({ ...values, pageSize: urlState.pageSize, sort: urlState.sort }))
```
Comment (frodare) [inline src/components/Filter.js:42]: This looks to be missing the deps list, also reformat like this for easier reading:
```
const process = useCallback(values => {
  updateUrl({ ...values, pageSize: urlState.pageSize, sort: urlState.sort }))
}, [urlState, updateUrl])
```

File: src/components/NoResults.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```
Comment (frodare) [inline src/components/NoResults.js:1]: Why was this added, was it to complement NoResultsMessage.js somehow?

**Review by frodare — CHANGES_REQUESTED** [review]

File: src/components/NoResults.js (hunk 2 above)
Comment (chauger) [reply src/components/NoResults.js:1]: No clue on this one.. I didn't touch that file..

Comment (frodare) [reply src/components/NoResults.js:1]: It was probably left over from the last time you worked on the project, but it needs to be removed from the PR so that it is not added when merged

**Review by frodare — APPROVED** [review]
I went ahead and removed the extra file and formatted the callback
